src/main/java/io/kestra/plugin/aws/eventbridge/PutEvents.java

```diff
@@ -0,0 +1,241 @@
+
+    @PluginProperty(dynamic = true)
+    @NotNull
+    @Schema(
+        title = "List of event entries to send to or internal storage uri to retrieve it.",
+        description = "A list of at least one EventBridge entry."
+    )
+    private Object entries;
+
+    @Override
+    public PutEvents.Output run(RunContext runContext) throws Exception {
+        final long start = System.nanoTime();
+
+        List<Entry> entryList = readEntryList(runContext, entries);
+
+        PutEventsResponse putEventsResponse = putEvents(runContext, entryList);
+
+        // Set metrics
+        runContext.metric(Timer.of("duration", Duration.ofNanos(System.nanoTime() - start)));
+        runContext.metric(Counter.of("failedEntryCount", putEventsResponse.failedEntryCount()));
+        runContext.metric(Counter.of("successfulEntryCount", entryList.size() - putEventsResponse.failedEntryCount()));
+        runContext.metric(Counter.of("entryCount", entryList.size()));
+
+        // Fail if failOnUnsuccessfulEvents
+        if (failOnUnsuccessfulEvents && putEventsResponse.failedEntryCount() > 0) {
+            var logger = runContext.logger();
+            logger.error("Response show {} event failed: {}", putEventsResponse.failedEntryCount(), putEventsResponse);
+            throw new RuntimeException(String.format("Response show %d event failed: %s", putEventsResponse.failedEntryCount(), putEventsResponse));
+        }
+
+        File tempFile = writeOutputFile(runContext, putEventsResponse, entryList);
+        return Output.builder()
+            .uri(runContext.putTempFile(tempFile))
+            .failedEntryCount(putEventsResponse.failedEntryCount())
+            .build();
+    }
+
+    private File writeOutputFile(RunContext runContext, PutEventsResponse putEventsResponse, List<Entry> entryList) throws IOException {
+        // Create Output
+        File tempFile = runContext.tempFile(".ion").toFile();
+        try (var stream = new FileOutputStream(tempFile)) {
+            List<PutEventsResultEntry> responseEntries = putEventsResponse.entries();
+            for (int i = 0; i < responseEntries.size(); i++) {
+                PutEventsResultEntry responseEntry = responseEntries.get(i);
+                OutputEntry entry = OutputEntry.builder()
+                    .entry(entryList.get(i))
+                    .eventId(responseEntry.eventId())
+                    .errorCode(responseEntry.errorCode())
+                    .errorMessage(responseEntry.errorMessage())
+                    .build();
+                FileSerde.write(stream, entry);
+            }
+        }
+        return tempFile;
+    }
+
+    private PutEventsResponse putEvents(RunContext runContext, List<Entry> entryList) throws IllegalVariableEvaluationException, JsonProcessingException {
+        try (var eventBridgeClient = client(runContext)) {
+            List<PutEventsRequestEntry> requestEntries = entryList.stream()
+                .map(entry -> {
```

Comment:
We have an helper for that `Rethrow.throwFunction()` usually statically imported